src/logging.c

```diff
@@ -0,0 +1,55 @@
+#include <stdarg.h>
+#include <time.h>
+
+#include "logging.h"
+
+
+/******************************************************
+ *** LOCAL DEFINES                                  ***
+ ******************************************************/
+
+/******************************************************
+ *** LOCAL VARIABLES                                ***
+ ******************************************************/
+
+static LoggingLevel min_dbg_lvl = FATAL;
```

Comment:
This variable should probably be called `min_log_lvl`

Reply:
Jup